apps/auth/db/db.py

```diff
@@ -12,10 +12,11 @@
 from flask import g
 from apps.auth.service import AuthService
 from superdesk import get_resource_service
-from apps.auth.errors import CredentialsAuthError, PasswordExpiredError
+from apps.auth.errors import CredentialsAuthError, PasswordExpiredError, ExternalUserError
 from flask import current_app as app
 from superdesk.utc import utcnow
 import datetime
+from superdesk.text_utils import get_text
```

Comment:
This is the incorrect version of `get_text` (this utility returns the plain text version of the provided HTML/XML value).

We need gettext to flag this error message for translation. On the server side this is done using the `flask_babel` module.

i.e.
```python
from flask_babel import _

def test():
    return _("Flags me for translation")
```